openpype/hosts/maya/plugins/publish/convert_gltf_shader.py

```diff
@@ -32,10 +32,12 @@ def process(self, instance):
                              glsl_shadingGrp + ".surfaceShader")
 
             # load the maya2gltf shader
-            maya_dir = os.getenv("MAYA_APP_DIR")
-            if not maya_dir:
-                raise RuntimeError("MAYA_APP_DIR not found")
-            ogsfx_path = maya_dir + "/maya2glTF/PBR/shaders/"
+            maya_publish = instance.context.data["project_settings"]["maya"]["publish"] # noqa
+
+            ogsfx_path = maya_publish["ConvertGLSLShader"]["ogsfx_path"]
+
+            if not os.path.exists(ogsfx_path):
+                raise RuntimeError("the ogsfx file not found")
```

Comment:
```suggestion
            ogsfx_path = instance.context.data["project_settings"]["maya"]["publish"]["ConvertGLSLShader"]["ogsfx_path"]  # noqa
            if not os.path.exists(ogsfx_path):
                raise RuntimeError("The ogsfx shader file does not exist: {}".format(ogsfx_path))
```
I'd at least report what path you've checked.
Also, if you using ` # noqa` anyway I'd just take the whole settings path on one line instead.